queries.sql

```diff
@@ -20,4 +20,91 @@ SELECT * FROM animals
 WHERE name != 'Gabumon';
 
 SELECT * FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+ALTER TABLE animals
+ADD COLUMN species VARCHAR(20);
```

Comment:
- [ ] Kindly make sure that Everything that creates or change the tables structure should be saved in the `schema.sql` file.

Reply:
Thanks